README.md

```diff
@@ -7,6 +7,8 @@ Welcome to the WordPress project documentation issue tracker. We're glad you're
 Please report issues here if your suggestion is related to the following
 
 1. Our development documentation located at https://developer.wordpress.org/ (Code named DevHub)
+   * The Description/Parameters/Return/Hooks sections of a Code Reference page are parsed and generated from the source code after each WordPress release. To make a change, please file a ticket on Core trac: https://core.trac.wordpress.org
```

Comment:
Maybe we could add here that focus should be `docs`.

Reply:
> Maybe we could add here that focus should be `docs`.

In my previous commits, I only mention the negatives ("what type of content you should not file against"), but not the positives because there are numerous.

I wonder if you can give an example of how you would amend the content. Should you mention Handbooks that reside in DevHub as well? but some of them link direct to the files on other GitHub repos (as if users should go there).

E.g. [Coding Standards](https://developer.wordpress.org/coding-standards/) "Handbook" links to [WordPress/wpcs-docs](https://github.com/WordPress/wpcs-docs)

Reply:
I wouldn't go that far for the readme file. We'll update contributing pages in docs Handbook for each part of documentation so we can add here link to that page. [This page](https://make.wordpress.org/docs/handbook/get-involved/current-docs-projects/) in particular is very much out of date but can be used for that purpose.

As for those GitHub repositories, Coding standards is not maintained by docs team so we won't document how to get involved there. The only such Handbook for now is Block editor one (which is shared with core team) and soon Advanced administration (shared with hosting team). Considering there is a different workflow for almost all handbooks I think it's best to dedicate them enough space in docs team handbook and use this readme more as a general guide with pointers for more info.

Reply:
I've included in a new commit that mentions the _positive_ aspects of the coverage such as Handbooks is listed separately.